packages/dht/src/connection/websocket/WebsocketServerConnection.ts

```diff
@@ -44,20 +41,21 @@ export class ServerWebsocket extends EventEmitter<ConnectionEvents> implements I
         this.socket = socket
     }
 
-    private onMessage(message: Message): void {
-        logger.trace('ServerWebsocket::onMessage')
-        if (message.type === MessageType.UTF8) {
-            logger.debug('Received string Message: ' + message.utf8Data)
-        } else if (message.type === MessageType.BINARY) {
-            logger.trace('Received Binary Message of ' + message.binaryData.length + ' bytes')
-            this.emit('data',
-                new Uint8Array(message.binaryData.buffer, message.binaryData.byteOffset,
-                    message.binaryData.byteLength / Uint8Array.BYTES_PER_ELEMENT))
+    private onMessage(message: WebSocket.RawData, isBinary: boolean): void {
+        if (!isBinary) {
+            logger.trace('Received string Message')
```

Comment:
Don't we emit any data from string messages? Is sending strings messages a supported use case or not?

Reply:
Would it be possible to close the WebSocket connection with an error message? And maybe write a log message which says explicitly that this is not supported.

Reply:
It could be possible and probably would make sense as it violates the protocol. However, I would leave it out of this PR.

Reply:
Could add that improvement to the "Future improvements" section.